azurerm/resource_arm_postgresql_configuration.go

```diff
@@ -55,6 +57,19 @@ func resourceArmPostgreSQLConfigurationCreateUpdate(d *schema.ResourceData, meta
 
 	value := d.Get("value").(string)
 
+	if requireResourcesToBeImported {
+		existing, err := client.Get(ctx, resGroup, serverName, name)
+		if err != nil {
+			if !utils.ResponseWasNotFound(existing.Response) {
+				return fmt.Errorf("Error checking for presence of existing PostgreSQL Configuration %s (resource group %s) ID", name, resGroup)
+			}
+		}
+
+		if existing.ID != nil && *existing.ID != "" {
+			return tf.ImportAsExistsError("azurerm_postgresql_configuration", *existing.ID)
+		}
+	}
```

Comment:
so the Configuration resources are a bit unique insofar as there's always going to be a default value (since this is effectively reading values from the `mysql.ini/pg.conf`), as such we need to treat it as a virtual resource (for example in the tests we reset the values back to the defaults rather than deleting them) - as such can we remove this? We've had to do the same for the MySQL Configuration resource

Once we flip this feature live, we'll probably want to add a disclaimer to the MySQL/PG Configuration pages that we're unable to require imports for them?